src/main/webapp/js/receipt-analysis.js

```diff
@@ -122,6 +122,11 @@ async function updateReceipt(event) {
   // Prevent the default action of reloading the page on form submission.
   event.preventDefault();
 
+  document.body.style.cursor = 'wait';
+  const saveButton = document.getElementById('submit-receipt');
+  const homeButton = document.getElementById('return-home');
+  saveButton.disabled = homeButton.disabled = true;
```

Comment:
Not sure if this is relevant since this isn't technically a declaration, but maybe this line should be separated. I also just think it would be better for readability

saveButton.disabled = true;
homeButton.disabled = true;

https://google.github.io/styleguide/jsguide.html#features-one-variable-per-declaration